solutions/cses-1132.mdx

```diff
@@ -2,60 +2,65 @@
 id: cses-1132
 source: CSES
 title: Tree Distances I
-author: Nathan Wang, Benjamin Qi, Abhishek Singh
+author: Nathan Wang, Benjamin Qi, Abhishek Singh, Brad Ma
 ---
 
 ## Solution 1
 
 Described in [CPH 14.3](https://usaco.guide/CPH.pdf#page=147).
 
+<LanguageSection>
+	
+<CPPSection>	
 ```cpp
-// source -> https://cses.fi/problemset/task/1132/
-
 #include<bits/stdc++.h>
 using namespace std;
 
 #define FIO ios::sync_with_stdio(0); cin.tie(0);
 
 
 vector<int> adj[200001];
-int d1[200001];  // to store first-max length.
-int d2[200001]; // to store second-max length.
-int c[200001]; // to store child for path of max length.
+int firstMax[200001];  // to store first-max length.
+int secondMax[200001];  // to store second-max length.
+int c[200001];   // to store child for path of max length.
```

Comment:
store 2e5 in a MAX_N
and then like use MAX_N + 1

Reply:
Up to Brad. a MAX_N would be nice, but this is fine too